acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -130,6 +143,25 @@ public void createAzureYamlFileAt(
     }
   }
 
+  public void createAzureCloudSigningYamlFileAt(
+      final Path metadataFilePath,
+      final String clientId,
+      final String clientSecret,
+      final String keyVaultName) {
+    try {
+      final Map<String, String> signingMetadata = new HashMap<>();
+      signingMetadata.put("type", "azure-cloud");
```

Comment:
Why do we need azure and azure-cloud file types?

Reply:
one peforms signing locally (bls or secp) the other does it in the cloud (secp only).